protostar/protostar_cli.py

```diff
@@ -80,8 +81,12 @@ def __init__(
         protostar_toml_writer: ProtostarTOMLWriter,
         protostar_toml_reader: ProtostarTOMLReader,
         requester: InputRequester,
+        start_time: float = 0.0,
```

Comment:
Why do we have to make this one being custom value? Can't we just run `= time.perf_counter()` in constructor?